h/services/group.py

```diff
@@ -34,21 +34,17 @@ def create_private_group(self, name, userid, description=None):
         :param name: the human-readable name of the group
         :param userid: the userid of the group creator
         :param description: the description of the group
+        :param type_flags: the type of this group
+        :param add_creator_as_member: if the group creator should be added as a member
```

Comment:
Oops, looks like these two params got documented in the wrong place, they're params of the `_create()` method below, not of this `_create_private_group()` method